app/models/contact.rb

```diff
@@ -5,19 +5,8 @@ class Contact < MailForm::Base
   def headers
     {
       :subject => "Connect Message from The Quad Co Site",
-      :to => "gsweitz1@jhu.edu",
+      :to => Secret.get('CONTACT_EMAIL'),
```

Comment:
Would changing this to ```Setting.contact_email``` allow for the specified email address to be changed in heroku with SendGrid?